src-ts/lib/profile-provider/profile-functions/profile.functions.ts

```diff
@@ -9,7 +9,11 @@ import { profileStoreGet, profileStorePatchName } from './profile-store'
 
 export async function getAsync(handle?: string): Promise<UserProfile | undefined> {
     handle = handle || (await tokenGetAsync())?.handle
-    return !handle ? Promise.resolve(undefined) : profileStoreGet(handle)
+    const output: UserProfile | undefined = !handle ? await Promise.resolve(undefined) : await profileStoreGet(handle);
+    // TODO: get real roles in PROD-2037
+    (output as UserProfile).isCustomer = true;
```

Comment:
Just so I understand, for now you're hard-coding the flags to indicate the user is both a customer and a member, so that we have access to both sets of tools? I see the //TODO so I'm assuming that's the case here, but perhaps add that to TODO?